test/JDBC/upgrade/latest/schedule

```diff
@@ -474,3 +474,4 @@ sys_certificates
 sys_database_permissions
 BABEL-4279
 BABEL-4484
+#AUTO_ANALYZE #uncomment this test when preparing for new minor version
```

Comment:
why is this commented? latest schedule is used for MVU, will it create an issue?

Reply:
For source_latest-target_latest path, upgrade of extensions will not happen as they are already on latest version so analyze will not run while PG will get upgraded.

Reply:
but analyze will run when we init the babelfish on latest. So, it should not matter if upgrade does not happen.

Reply:
Since we upgraded the PG database in that case we will recreated the table and inserted the values so the affect of analyze will not there.